docs/technical_reference/property_models/ADM1.rst

```diff
@@ -0,0 +1,205 @@
+ADM1 Property Package
+=====================
+
+This package implements properties and reactions of an anaerobic digestion model for industrial wastewater treatment from food and beverage production using an anaerobic digester as provided in
+`Batstone, D. J. et al. (2002) <https://iwaponline.com/wst/article-abstract/45/10/65/6034>`_ and `Rosen and Jeppsson (2006) <https://www.iea.lth.se/WWTmodels_download/TR_ADM1.pdf>`_.
+
+This Anaerobic Digestion Model no.1 (ADM1) property/reaction package:
+   * supports 'H2O', 'S_su', 'S_aa', 'X_fa', 'X_va', 'X_bu', 'X_pro', 'X_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', and 'S_co2' as components
+   * supports only liquid and vapor phase
+
+Sets
+----
+.. csv-table::
+  :header: "Description", "Symbol", "Indices"
+
+  "Components", ":math:`j`", "['H2O', 'S_su', 'S_aa', 'X_fa', 'X_va', 'X_bu', 'X_pro', 'X_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', 'S_co2']"
+  "Phases", ":math:`p`", "['Liq', 'Vap']"
+
+Components
+----------
+.. csv-table::
+  :header: "Description", "Symbol", "Variable"
+
+  "Monosaccharides, S_su", ":math:`S_su`", "S_su"
+  "Amino acids, S_aa", ":math:`S_aa`", "S_aa"
+  "Long chain fatty acids, S_fa", ":math:`S_fa`", "S_fa"
+  "Total valerate, S_va", ":math:`S_va`", "S_va"
+  "Total butyrate, S_bu", ":math:`S_bu`", "S_bu"
+  "Total propionate, S_pro", ":math:`S_pro`", "S_pro"
+  "Total acetate, S_ac", ":math:`S_ac`", "S_ac"
+  "Hydrogen gas, S_h2", ":math:`S_{h2}`", "S_h2"
+  "Methane gas, S_ch4", ":math:`S_{ch4}`", "S_ch4"
+  "Inorganic carbon, S_IC", ":math:`S_IC`", "S_IC"
+  "Inorganic nitrogen, S_IN", ":math:`S_IN`", "S_IN"
+  "Soluble inerts, S_I", ":math:`S_I`", "S_I"
+  "Composites, X_c", ":math:`X_c`", "X_c"
+  "Carbohydrates, X_ch", ":math:`X_ch`", "X_ch"
+  "Proteins, X_pr", ":math:`X_pr`", "X_pr"
+  "Lipids, X_li", ":math:`X_li`", "X_li"
+  "Sugar degraders, X_su", ":math:`X_su`", "X_su"
+  "Amino acid degraders, X_aa", ":math:`X_aa`", "X_aa"
+  "Long chain fatty acid (LCFA) degraders, X_fa", ":math:`X_fa`", "X_fa"
+  "Valerate and butyrate degraders, X_c4", ":math:`X_c4`", "X_c4"
+  "Propionate degraders, X_pro", ":math:`X_pro`", "X_pro"
+  "Acetate degraders, X_ac", ":math:`X_ac`", "X_ac"
+  "Hydrogen degraders, X_h2", ":math:`X_{h2}`", "X_h2"
+  "Particulate inerts, X_I", ":math:`X_I`", "X_I"
+  "Total cation equivalents concentration, S_sat", ":math:`S_sat`", "S_sat"
```

Comment:
```suggestion
  "Total cation equivalents concentration, S_cat", ":math:`S_{cat}`", "S_cat"
```
This should be `S_cat` where "cat" stands for "cations". Does the model list `S_sat` instead of `S_cat`? If so, though it seems trivial, we should point that out on #857 .